parm/gfs/biascorr_atms_n20.yaml

```diff
@@ -0,0 +1,100 @@
+diagnostics:
+
+# Data read
+# ---------
+datasets:
+  - name: bcoef
+    satellite: atms
+    sensor: n20
+    type: RadDataSpace
+    filename: /scratch1/NCEPDEV/da/Kevin.Dougherty/data_processing_tool/data/diag_atms_n20_2021032500.nc
+    channels: &chans [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]
+    variable: &var brightnessTemperature
+    qc variable: &qcvar EffectiveQC0
+    outfile: /scratch1/NCEPDEV/da/Kevin.Dougherty/outfile.nc
+    variables to process:
+    - function: calculate_omf
+      loops: [0, 1]
+      variable: *var
+      channels: *chans
+      qc var: *qcvar
```

Comment:
I don't think you need qcvar anymore because of the `loops` but need to double check my logic on that.